.github/workflows/scripts/workflow.js

```diff
@@ -6,6 +6,11 @@ module.exports = async ({ core, context, fetch }, token) => {
   const repo = 'charts'
   const ref = 'master'
   const workflow_name = 'Push'
+  let version = context.payload.workflow_run.head_branch || context.payload.ref
+  if (context.payload.workflow_run.head_branch) {
+    version = `refs/tags/${version}`;
+  }
```

Comment:
I _think_ this may not work if `context.payload.workflow_run` is `undefined`.
I would suggest this code instead:
```suggestion
  let version = context.payload.ref
  if (context.payload.workflow_run !== undefined) {
    version = `refs/tags/${context.payload.workflow_run.head_branch}`
  }
```

Reply:
Excellent, thanks for the thorough testing @ljubon!